vcf2fhir/fhir_helper.py

```diff
@@ -488,72 +484,31 @@ def generate_final_json(self):
                 fhirReport['derivedFrom'] = derivedFrom
 
         for k, i in enumerate(od['contained']):
+            od_contained_k = od['contained'][k]
+            v_c_c = 'valueCodeableConcept'

```

Comment:
Ya I was hoping I could do that but 
then 
```

 od_contained_k_component_q['valueCodeableConcept']['coding'][0] =\<- is again above the limit 
                            get_key_dict(j, v_c_c)
                                            

od_contained_k_component_q\ <- it giving some pep8 warning(PEP 8: E211 whitespace before '[')
                 ['valueCodeableConcept']['coding'][0] =\
                            get_key_dict(j, v_c_c)
                            
```
so to handle that I had to make a variable to store 'valueCodeableConcept ' so I thought why not just declare it above and use again and again 

please suggest to me how should I process further

Reply:
This 79 character limit is bad. I think, lot can be simplified in the existing code. We may not have to iterate dict like we are doing today.  Something like below can be used. 
```
object = { valueCodeableConcept : { coding : [ {}]}}
key_path = 'valueCodeableConcept.coding.0'

def get_object_at_path(object, path):
      temp = object
      for key in path.split('.')
           try: 
                key = int(key)
           except _:
                pass
           if temp[key]:
                temp = temp[key]
           else: 
                // path does not exist
                return None
     return temp
```
However I am wondering why we are just converting a dictionary and appending in same object. I would have to do some experiment to find a better approach.

Reply:
Ya exactly I was also thinking the same approach then but then I thought it would bad approach to do it in for loop also for which that can be accessed directly.

I was hoping we could do something like that

```
path=["valueCodeableConcept","coding",0]

def get_object_at_path(object, path):
      temp = object
      for key in path:
           if temp[key]:
                temp = temp[key]
           else: 
                return None
     return temp
```
And also I am sorry I forgot about the test case and I am pushing that test case also

Reply:
@srgothi92 Just a suggestion, can we just increase the max length limit of character in a line to 120.